lib/w_common.dart

```diff
@@ -12,7 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-export 'disposable.dart' show Disposable;
```

Comment:
Not needed now -- we decided this will continue to be the non-browser specific entry point.